metricbeat/values.yaml

```diff
@@ -141,22 +140,26 @@ labels: {}
 managedServiceAccount: true
 
 clusterRoleRules:
-  - apiGroups:
-    - "extensions"
-    - "apps"
-    - ""
-    resources:
-    - namespaces
-    - pods
-    - events
-    - deployments
-    - nodes
-    - nodes/stats
-    - replicasets
-    verbs:
-    - get
-    - list
-    - watch
+- apiGroups: [""]
+  resources:
+  - nodes
+  - namespaces
+  - events
+  - pods
+  verbs: ["get", "list", "watch"]
+- apiGroups: ["extensions"]
+  resources:
+  - replicasets
+  verbs: ["get", "list", "watch"]
+- apiGroups: ["apps"]
+  resources:
+  - statefulsets
+  - deployments
```

Comment:
We also need `replicasets` here. There was an issue with this: https://github.com/elastic/beats/pull/15741

Reply:
fixed in https://github.com/elastic/helm-charts/pull/567/commits/470fecdc483f120f80f60cca68f812656e8fb381